Create ui-c8y-1022-21-1-added-fix-for-chart-timestamp-when-english-us-is-selected.md (#3649)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1022-21-1-added-fix-for-chart-timestamp-when-english-us-is-selected.md

---
date: ""

> title: Fixed data point graph and data explorer chart timestamps when English US locale is selected

Reviewer: ```suggestion
title: Fixed data point graph and data explorer chart timestamps for English US locale
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64163
version: 1022.21.1
---

> In some scenarios, when the English US language was selected in the application, the data point graph and data explorer chart timestamps were incorrectly displayed by showing 24 hour format instead of 12 hour format. This issue has been resolved and the chart timestamps are now correctly shown when using the English US locale. This change ensures a consistent user experience across different locales.

Reviewer: ```suggestion
In some scenarios, when the English US language was selected in the platform, the data point graph and data explorer chart timestamps were incorrectly displayed, showing a 24-hour format instead of a 12-hour format. This issue has been resolved. The chart timestamps are now correctly shown when using the English US locale. This change ensures a consistent user experience across different locales.
```

---
date: ""

> title: Added fix for chart timestamp when English US is selected.

Reviewer 2: ```suggestion
title: Fixed data point graph and data explorer chart timestamps when English US locale is selected
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64163
version: 1022.21.1
---

> Co-authored-by: Enio Sultan <92023325+eniosultan@users.noreply.github.com>
> Co-authored-by: Lukasz Janusz <114157358+luja-c8y@users.noreply.github.com>

Reviewer 2: ```suggestion
In some scenarios, when the English US language was selected in the application, the data point graph and data explorer chart timestamps were incorrectly displayed by showing 24 hour format instead of 12 hour format. This issue has been resolved and the chart timestamps are now correctly shown when using the English US locale. This change ensures a consistent user experience across different locales.
```
